### PR TITLE
Helm services

### PR DESCRIPTION
**Pull request by jimmykarily**
Docs for https://github.com/epinio/epinio/issues/1332

### CODE REVIEW

File: src/howtos/create_custom_service.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+```
+$ kubectl get services.application.epinio.io -n epinio mysql-dev -o yaml > service.yaml
+```
+
+or you can find the definition of the catalog services here:
+
+https://github.com/epinio/helm-charts/blob/3a12bac7aee5ac36c6d43416f2e83ac10090c62a/chart/epinio/templates/service-catalog.yaml
+
+Change the fields too point to the desired helm chart and apply the yaml with a command like:
```
Comment (juadk) [inline src/howtos/create_custom_service.md:21]:
```suggestion
Change the fields to point to the desired helm chart and apply the yaml with a command like:
```

File: src/howtos/create_custom_service.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+$ kubectl get services.application.epinio.io -A
+```
+
+If Epinio is installed in a namespace called `epinio`, the following command should give you a yaml to work on:
+
+```
+$ kubectl get services.application.epinio.io -n epinio mysql-dev -o yaml > service.yaml
+```
+
+or you can find the definition of the catalog services here:
```
Comment (juadk) [inline src/howtos/create_custom_service.md:17]:
```suggestion
or you can find the definition of the catalog services [here](https://github.com/epinio/helm-charts/blob/3a12bac7aee5ac36c6d43416f2e83ac10090c62a/chart/epinio/templates/service-catalog.yaml
).
```

File: src/references/services.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+# Services
+
+Most real world applications work together with other services to store data, logs, to exchange messages with other systems etc.
+The most common example is probably a database. There are at least 2 ways to connect an Epinio application to a service.
+The first one is to provision the service manually and [create an Epinio Configuration](configurations.md) to make the service connection details
+available to the application.
+
+The second one lets Epinio provision the service which you can later bind to your application with the `epinio service bind` command. This page explains how this works.
+
+Services in Epinio are managed by the [epinio service](cli/epinio_service.md) commands.
```
Comment (juadk) [inline src/references/services.md:10]: I got 404 here, I think you forgot to add the link into the SUMMARY.

**Review by juadk — CHANGES_REQUESTED** [review]
I found broken link and one typo, LGTM for the rest.

File: src/howtos/create_custom_service.md (hunk 2 above)
Comment (juadk) [reply src/howtos/create_custom_service.md:17]: FMPOV, more readable with the link hidden.

File: src/howtos/create_custom_service.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Create a custom service
+
+As described [in the Service reference page](../references/services.md), Epinio services are nothing but descriptions of Helm charts. An operator may add more Service by simply creating a Custom Resource in the namespace where Epinio is installed.
```
Comment (manno) [inline src/howtos/create_custom_service.md:3]:
```suggestion
As described [in the Service reference page](../references/services.md), Epinio services are nothing but descriptions of Helm charts. An operator may add more services by simply creating a Custom Resource in the namespace where Epinio is installed.
```

**Review by manno — APPROVED** [review]

File: src/references/services.md (hunk 3 above)
Comment (juadk) [reply src/references/services.md:10]: You can add this in the SUMMARY just under the configurations part:

```
      - [epinio_service](references/cli/epinio_service.md)

        - [epinio_service_bind](references/cli/epinio_service_bind.md)
        - [epinio_service_catalog](references/cli/epinio_service_catalog.md)
        - [epinio_service_create](references/cli/epinio_service_create.md)
        - [epinio_service_delete](references/cli/epinio_service_delete.md)
        - [epinio_service_list](references/cli/epinio_service_list.md)
        - [epinio_service_show](references/cli/epinio_service_show.md)
```

Comment (jimmykarily) [reply src/references/services.md:10]: Right! Fixed in d783815

**Review by juadk — APPROVED** [review]

File: src/references/services.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  Namespace: workspace
+  Application: sample
+  cnb@sample-7bc756989-tjvt9:/$ cat /configurations/workspace-mydb-mysql/
+  ...
+  cnb@sample-7bc756989-tjvt9:/$ exit
+  ```
+
+At this point, it's up to the application to make use of that secret. The helm chart that is used for this example, doesn't write the hostname of the database in the generated secret. That's because the Helm chart wasn't created specifically for Epinio. To figure out the hostname in this case, requires some knowledge on [how Kubernetes DNS works](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/).
+
+In this case it is: `workspace-mydb-mysql.workspace.svc.cluster.local`.
```
Comment (nunix) [inline src/references/services.md:67]:
```suggestion
For the example above, the hostname is: `workspace-mydb-mysql.workspace.svc.cluster.local`.
```

File: src/references/services.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+
+When you are done with this example you can delete all generated resources:
```
Comment (nunix) [inline src/references/services.md:70]:
```suggestion
When you are done with this example, you can delete all generated resources with the following commands:
```

**Review by nunix — COMMENTED** [review]
Rephrased to active voice and made spelling modifications.

**Pull request merged**